src/SigV4MessageHandler/SigV4MessageHandler.cs

```diff
@@ -0,0 +1,232 @@
+using System.Globalization;
```

Comment:
we might need to include the Amazon header to each of the files we copied from the SDK.